pkg/metrics/grafana/tidb.json

```diff
@@ -15083,23 +15083,23 @@
           "steppedLine": false,
           "targets": [
             {
-              "expr": "histogram_quantile(0.999, sum(rate(tidb_ddl_scan_rate_bucket{k8s_cluster=\"$k8s_cluster\", tidb_cluster=~\"$tidb_cluster.*\", instance=~\"$instance\"}[1m])) by (le, type))",
```

Comment:
maybe add a rule to 
https://github.com/pingcap/tidb/blob/master/tools/dashboard-linter/main.go

Reply:
https://github.com/pingcap/tidb/pull/54135/commits/4383bde34f0704fd432dd35523612aad33d936a5